pages/en/zkapps/how-to-test-a-zkapp.mdx

```diff
@@ -37,8 +37,97 @@ Because we are using Jest, it's helpful to break down all functionality of your
 
 For examples of existing tests, we recommend creating a template example using the Mina zkApp CLI via `zk project <name>` and examining the test file there. [You will see a basic example of a few tests](https://github.com/o1-labs/zkapp-cli/blob/main/templates/project-ts/src/Add.test.ts) that deploy and update the state on a smart contract using Jest.
 
+#### Creating a local blockchain
+
+The `Mina.LocalBlockchain()` method specifies a mock Mina ledger of accounts that runs locally. It contains logic for updating the ledger that can be used to test your smart contract locally.
+
+```ts
+let Local = Mina.LocalBlockchain();
+Mina.setActiveInstance(Local);
+```
+
+#### Deploying a contract locally
+
+The mock Mina local blockchain contains test accounts that can be used to deploy the contract and pay transaction fees.
```

Comment:
This might read more clearly if the steps were connected together somehow--e.g "First, "; "Next, "; "Then" (or similar), to help walk them through it as instructions.

Reply:
Good idea.